Scenarios.cs

```diff
@@ -3,7 +3,8 @@
     internal enum Scenarios
     {
         WikiCohereEnglishEmbeddingOnly,
-        WikiCohereEnglishEmnbeddingOnlyStreaming
+        WikiCohereEnglishEmnbeddingOnlyStreaming,
+        MSTuringEmbeddingOnly

```

Comment:
If this class works for both MSTuring and MSMarco can we have the name reflect that?